src/sections/Pricing/generateOptions.js

```diff
@@ -45,14 +44,12 @@ function generateOptions(data) {
     },
   };
 
-  console.log("Tiers configuration:", tiers);
-
   const options = Object.entries(tiers).map(([tierName, tierInfo]) => {
-    console.log(`Processing tier: ${tierName}`, tierInfo);
 
     const summary = data
       .filter((item) => {
         const matches = item.subscription_tier === tierName && (item.pricing_page === "true" || item.pricing_page === "X" ) ;
```

Comment:
```suggestion
        const matches = item.subscription_tier === tierName && (item.pricing_page === "true" || item.pricing_page === "x" ) ;
```

At this point in the flow, I believe that we've shifted to lowercase (from the prior transformation in the workflow). Given that prudence we can skip another .toLowercase(), but do need to use a lowercase "x" in comparison.